extractors/nerdextractor.py

```diff
@@ -0,0 +1,45 @@
+import html
+from pathlib import Path
+from collections import namedtuple
+
+import requests
+from bs4 import BeautifulSoup
+
+Icon = namedtuple('Icon', 'icon name')
+
+class NerdExtractor(object):
+    def __init__(self: 'NerdExtractor'):
+        self.icons = []
+
+    def fetch_icons(self: 'NerdExtractor'):
+        print('Downloading list of Nerd Font icons')
+
+        response = requests.get(
+            'https://www.nerdfonts.com/cheat-sheet',
+            timeout=60
+        )
+
+        characters = BeautifulSoup(response.content.decode(response.encoding), 'html.parser').find(id='glyphCheatSheet').find_all(class_='column')
```

Comment:
I think you can use `response.text` instead of manually decoding it.

Reply:
That works too, I was just reusing code from another extractor.